javascript/javascript2/week1/find-SmallerWord.js

```diff
@@ -0,0 +1,23 @@
+let danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+function findSmallerWord(danishWords)
+{
+    let wordLength;
+    let wordIndex;
+    let minWordLength = 256;  // choose a max number
+    for (let i = 0; i < danishWords.length; i++)
+    {
+        wordLength = danishWords[i].length;
+        
+        if(wordLength < minWordLength)
+        {
+            minWordLength = wordLength;  // store the minimum length
+            wordIndex = i; // store the index of the smaller word
+        }
+    }
+  
+    console.log(danishWords[wordIndex]);
+    console.log(minWordLength);
+
+    return ' ';
```

Comment:
You don't have to return anything in this case. It's okay to have a method that is 'not returning anything'